### DISCUSSION TITLE
How do I confirm that I am using self host

### DISCUSSION THREAD

**Discussion by weironz** [rustdesk/rustdesk#7473] [Q&A]
I install rustdesk in aliyun ECS with docker
```
root@ubuntu:~# docker ps |grep hbb
da517850b049   rustdesk/rustdesk-server:1.1.6   "hbbr"                   28 minutes ago   Up 28 minutes   0.0.0.0:21117->21117/tcp, :::21117->21117/tcp, 0.0.0.0:21119->21119/tcp, :::21119->21119/tcp                                                                          hbbr
56816b8c778f   rustdesk/rustdesk-server:1.1.6   "hbbs -r hbbr:21117"     32 minutes ago   Up 32 minutes   0.0.0.0:21115-21116->21115-21116/tcp, :::21115-21116->21115-21116/tcp, 0.0.0.0:21118->21118/tcp, :::21118->21118/tcp, 0.0.0.0:21116->21116/udp, :::21116->21116/udp   hbbs
root@ubuntu:~# 
```

then i config rustdesk client

![image](https://github.com/rustdesk/rustdesk/assets/33590311/ec119a6f-0dc0-49ba-a7e1-fbe08ee904aa)

It say success, is this mean my self-host setting is ok ,and connection is ok?

Then how do I confirm that I am using self host with remote connection?

Is there a hint that you are using a self built relay to connect to a remote desktop instead of the official relay server provided

**Comment by xlionjuan** [comment]
Success 成功 means you have saved the settings


Not mean it is connected
And you should use key to connect to your server for better security
